### ISSUE TITLE
Relationships are not rendered for Postgres database 

### ISSUE THREAD

**Issue by dannief** [sue445/plant_erd#179]
I have a Postgres database with tables across 4 schemas including public. All the tables are rendered, but no relationship between tables

**Comment by sue445** [comment]
@dannief Please post the schema information (which including table names, column names and foreign keys), as that information is not enough to tell me anything.

**Comment by dannief** [comment]
Ran the following command: (See database schema and result below)
`./plant_erd postgresql --host localhost --port 5432 -database test --user postgres --password thepassword --file test-db.puml`

```sql
-- DROP SCHEMA people;

CREATE SCHEMA people AUTHORIZATION postgres;
-- people.author definition

-- Drop table

-- DROP TABLE people.author;

CREATE TABLE people.author (
	id int4 NOT NULL GENERATED BY DEFAULT AS IDENTITY,
	"name" varchar NOT NULL,
	CONSTRAINT author_pk PRIMARY KEY (id)
);


-- people.book_author definition

-- Drop table

-- DROP TABLE people.book_author;

CREATE TABLE people.book_author (
	isbn varchar NOT NULL,
	author_id int4 NOT NULL,
	CONSTRAINT book_author_pk PRIMARY KEY (isbn, author_id),
	CONSTRAINT book_author_fk_1 FOREIGN KEY (author_id) REFERENCES people.author(id) ON DELETE CASCADE ON UPDATE CASCADE
);

-- DROP SCHEMA "library";

CREATE SCHEMA "library" AUTHORIZATION postgres;
-- "library".book definition

-- Drop table

-- DROP TABLE "library".book;

CREATE TABLE "library".book (
	isbn varchar NOT NULL,
	"name" varchar NOT NULL,
	CONSTRAINT book_pk PRIMARY KEY (isbn)
);
```

```plantuml
entity author {
  * id : integer
  --
  * name : character varying
}

entity book {
  * isbn : character varying
  --
  * name : character varying
}

entity book_author {
  * isbn : character varying
  * author_id : integer
}
```

**Comment by sue445** [comment]
@dannief Thank you for reporting.

I fixed at #182 and released v0.4.1 🚀 

* https://github.com/sue445/plant_erd/blob/master/CHANGELOG.md#v041
* https://github.com/sue445/plant_erd/releases/tag/v0.4.1

```
entity library.book {
  * isbn : character varying
  --
  * name : character varying
}

entity people.author {
  * id : integer
  --
  * name : character varying
}

entity people.book_author {
  * isbn : character varying
  * author_id : integer
}

people.book_author }-- people.author
```

![plant_erd_issue_179](https://user-images.githubusercontent.com/608755/164914742-63a08e99-63a7-4b6a-80e6-998747cf894b.svg)

**Comment by dannief** [comment]
Thank you.
